src/value.rs

```diff
@@ -179,6 +183,33 @@ impl<'t, S> Column<'t, S, i64> {
     }
 }
 
+impl<'t, S, T: LikeTyp + 't> Column<'t, S, T> {
+    /// Check if the column starts with a string pattern.
+    pub fn starts_with(&self, pattern: impl Into<String>) -> Column<'t, S, bool> {
+        Like(self, format!("{}%", pattern.into())).into_column()
+    }
+
+    /// Check if the column ends with a string pattern.
+    pub fn ends_with(&self, pattern: impl Into<String>) -> Column<'t, S, bool> {
+        Like(self, format!("%{}", pattern.into())).into_column()
+    }
+
+    /// Check if the column contains a string pattern.
+    pub fn contains(&self, pattern: impl Into<String>) -> Column<'t, S, bool> {
+        Like(self, format!("%{}%", pattern.into())).into_column()
+    }
+
+    /// Check if the column matches to a pattern [pattern ref](https://www.w3schools.com/sql/sql_like.asp).
+    pub fn like(&self, pattern: impl Into<String> + Clone + 't) -> Column<'t, S, bool> {
+        Like(self, pattern).into_column()
+    }
```

Comment:
It would be nice to link to the official documentation instead of w3schools, even though that documentation is very little https://www.sqlite.org/lang_expr.html#like
You could also mention that `like` is case insensitive and that it has two special characters (`_` and `%`).
Then you can also mention that you chose `\` as the escape character.